packages/ckeditor5-html-support/tests/htmlcomment.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import { getData as getModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+
+import HtmlComment from '../src/htmlcomment';
+
+describe( 'HtmlComment', () => {
+	let editor, model, root;
+
+	beforeEach( () => {
+		return VirtualTestEditor
+			.create( {
+				plugins: [ HtmlComment, Paragraph ]
+			} )
+			.then( newEditor => {
+				editor = newEditor;
+				model = editor.model;
+				root = model.document.getRoot();
+
+				model.schema.register( 'div' );
+				model.schema.extend( '$block', { allowIn: 'div' } );
+				model.schema.extend( 'div', { allowIn: '$root' } );
+				model.schema.extend( 'div', { allowIn: 'div' } );
+
+				editor.conversion.elementToElement( { model: 'div', view: 'div' } );
+			} );
+	} );
+
+	afterEach( () => {
+		return editor.destroy();
+	} );
+
+	it( 'should be named', () => {
+		expect( HtmlComment.pluginName ).to.equal( 'HtmlComment' );
+	} );
+
+	it( 'should be loaded', () => {
+		expect( editor.plugins.get( HtmlComment ) ).to.be.instanceOf( HtmlComment );
+	} );
```

Comment:
I know that these tests are present in all our plugins but they are quite dumb TBH :smile: 

Perhaps we could replace these two tests with the following: (?)

```
it( 'should be loadable using its plugin name', () => {
	expect( editor.plugins.get( 'HtmlComment' ) ).to.be.instanceOf( HtmlComment );
} );
```

Reply:
Yeah, good idea.
Changed as proposed.